<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ReportBut">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuizBut">
   <property name="geometry">
    <rect>
     <x>629</x>
     <y>105</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quiz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExamBut">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>105</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exam papers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TopicFilterBut">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>485</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Topic Filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BlogBut">
   <property name="geometry">
    <rect>
     <x>639</x>
     <y>485</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Blog</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignOutBut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
